pandas/core/generic.py

```diff
@@ -2146,8 +2147,10 @@ def tail(self, n=5):
         Returns last n rows
         """
         l = len(self)
-        if l == 0 or n == 0:
+        if l == 0:
```

Comment:
So why was it there in the first place?  I don't really understand what its doing in either head() or tail().